app/lua53/host/luac.c

```diff
@@ -18,38 +18,35 @@
 #include <time.h>
 
 #include "lua.h"
-#include "lualib.h"
-#include "lauxlib.h"
-#include "ldebug.h"
+#include "lualib.h"       // ??
+#include "ldebug.h"       // ??
 #include "lnodemcu.h"
 #include "lobject.h"
 #include "lstate.h"
-#include "lstring.h"
+#include "lstring.h"      // ???
```

Comment:
Johny, in terms of this and all of your `luac.c` comments, I get the impression that the original `luac.c` was a "throw-in" not developed by the original Lua developers.  The coding style and layout is quite different to the core Lua code. We are getting to the stage where this is heavily modified because of our LFS, Absolute LFS and execution environment support.  I've pretty much decided to give this a complete hose down starting with a pretty print reformat, and getting rid of the some of the coding nasties.  I'll add this to the next push.  I will also reflect your general comments.